### DISCUSSION TITLE
Can I have other file instead of conftest.py?

### DISCUSSION THREAD

**Discussion by ssandr1kka** [ElSnoMan/pyleniumio#168] [Q&A] [answered]
Hi, I have gone through the documentations.
In general I think you have made an amazing feature, with really good chance of growth, so good luck!
I am working on large app where conftest is already taken by other testing methods, there is no way I can remove, replace or rename it. I am sure that will mess all pipeline. 
It would be easier if I could have some other file for configuration pylenium. Is that possible? How can I do so?
Once again, great lib, nice intro video, good luck, hope to see this reaching as high as cypress and even more haha

**Comment by ElSnoMan** [comment, marked as answer]
The fixtures within conftest.py are required for Pylenium to function as expected. However, they are just fixture functions, so you can just copy and paste them into your existing conftest.py at the Project Root
